ui/src/test/org/jboss/windup/web/selenium/Selenium04Test.java

```diff
@@ -0,0 +1,224 @@
+package org.jboss.windup.web.selenium;
+
+import java.awt.AWTException;
+import java.util.ArrayList;
+
+import junit.framework.TestCase;
+
+public class Selenium04Test extends TestCase {
+
+	private AnalyzeProject selenium;
+
+	public void setUp() throws InterruptedException {
+		selenium = new AnalyzeProject();
+	}
+
+	public void testStep01() throws InterruptedException, AWTException {
+		/*
+		 * Step 1
+		 */
+		assertEquals("Application List", selenium.headerTitle());
+		assertEquals("Application List", selenium.pageTitle());
+
+		selenium.switchTab(2);
+		assertEquals("All Issues", selenium.pageTitle());
+
+		selenium.switchTab(3);
+		assertEquals("Technologies", selenium.pageTitle());
+
+		selenium.switchTab(4);
+		assertEquals("Dependencies", selenium.pageTitle());
+
+		selenium.switchTab(5);
+		assertEquals("About", selenium.pageTitle());
+
+		selenium.switchTab(1);
+		selenium.clickSendFeedback();
+		/*
+		 * Step 2
+		 */
+		selenium.closeFeedback();
+		selenium.closeDriver();
+	}
+
+	public void testAppList() throws AWTException {
+		assertEquals("Application List", selenium.headerTitle());
+		assertEquals("Application List", selenium.pageTitle());
+
+		// Steps 3 - 4
+		assertTrue(selenium.applicationSort());
+
+		// Step 5
+		selenium.filterAppList("Name", "ad");
+		assertEquals("[AdditionWithSecurity-EAR-0.01.ear, AdministracionEfectivo.ear]", selenium.listApplications().toString());
+
+		// Step 6
+		selenium.filterAppList("Tag", "JPA XML 2.0");
+		assertEquals("[AdministracionEfectivo.ear]", selenium.listApplications().toString());
+
+		// Step 7
+		selenium.changeRelationship("Matches any filter (OR)");
+		assertEquals("[AdditionWithSecurity-EAR-0.01.ear, AdministracionEfectivo.ear]", selenium.listApplications().toString());
+		
+		// Step 8
+		selenium.clearFilters();
+		assertEquals("[AdditionWithSecurity-EAR-0.01.ear, arit-ear-0.8.1-SNAPSHOT.ear, AdministracionEfectivo.ear]",
+				selenium.listApplications().toString());
+
+		// Step 9
+		selenium.filterAppList("Tag", "JPA XML 2.0");
+		assertEquals("[AdministracionEfectivo.ear]", selenium.listApplications().toString());
+		
+		// Step 10
+		assertTrue(selenium.deleteFilter("Tag: ", "JPA XML 2.0"));
+		assertEquals("[AdditionWithSecurity-EAR-0.01.ear, arit-ear-0.8.1-SNAPSHOT.ear, AdministracionEfectivo.ear]",
+				selenium.listApplications().toString());
+
+		selenium.closeDriver();
+	}
+
+	public void testStep02() throws InterruptedException
+	{
+
+		selenium.switchTab(2);
+		
+		// Step 11
+		assertEquals("All Issues", selenium.pageTitle());
+		assertEquals("[Migration Optional, Cloud Mandatory, Cloud Optional]", selenium.allIssuesReport().toString());
```

Comment:
After we merged https://github.com/windup/windup/pull/1331, `Information` has to be added for this test to work.
Other than this, anyway, better comparing `List`s using the `equals` method and not they're "string" version.

Reply:
done